src/components/Button.js

```diff
@@ -79,6 +91,7 @@ const Button = (props) => {
             disabled={props.isLoading || props.isDisabled}
             style={[
                 ...additionalStyles,
+                props.shouldRemoveRightBorderRadius ? styles.flex1 : undefined,
```

Comment:
The other uses of these` props make sense, but this one is a little confusing to me. Why is `flex1` only necessary on the **right** prop, instead of **left** AND/OR **right**?

Is it something to do with left alignment being the default?

Reply:
This can maybe be improved. The idea here was that if we have a button with a flat right side then something is positioned to the right of it and it's probably in a containing element with `flexDirection: 'row'`. In those cases, the `flex: 1` should be applied so the "button" (left element) expands to take up the entire area (which it will only do when `flex: 1` is applied.

Another option here would be to just pass this in from the outside on `additionalStyles` so I'll look into that instead..